src/core/structure/cellStyle.ts

```diff
@@ -31,6 +31,25 @@ export default class CellStyle extends Cloneable<CellStyle> {
     return this;
   }
 
+  applyCss(css: Map<string, string>): CellStyle {
+    // If a style is set in other but not in this, apply it to this.
+    for (const [key, value] of css) {
+      if (!this.styling.has(key)) {
+        this.styling.set(key, value);
+      }
+    }
+
+    return this;
+  }
+
+  clearCss() {
+    this.styling = new Map();
```

Comment:
Since the PR refers to the styling map as css, the field name should be changed as well.

Reply:
Did it but I dont like to use the word css either, but cant think of anything else. Do let me know if any other name comes to your mind